packages/p2p/src/components/modal-manager/modals/disclaimer-modal/disclaimer-modal.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { Button, Checkbox, Icon, Modal, Text } from '@deriv/components';
+import { useStore } from '@deriv/stores';
+import { Localize } from 'Components/i18next';
+import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
+
+const ModalTitle = () => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
+    return (
+        <React.Fragment>
+            <Icon icon='IcWarning' size={64} className='disclaimer-modal__body-icon' />
+            <Text
+                as='p'
+                color='prominent'
+                weight='bold'
+                line_height={is_mobile ? 'xl' : 'xxl'}
+                size={is_mobile ? 'xs' : 's'}
+            >
+                <Localize i18n_default_text='For your safety:' />
+            </Text>
+        </React.Fragment>
+    );
+};
+const getDislaimerStatements = () => [
+    <Localize
+        i18n_default_text='If you’re selling, only release funds to the buyer after you’ve received payment.'
+        key={0}
```

Comment:
Why you are adding keys to the Localize?

Reply:
there was some eslint error.. failed in precommit hook. add key prop

Reply:
What was the error?

Reply:
missing key element in iterator. even after giving the key to the li wrapper.

Reply:
Can u please remove it and try that one more time
I guess this time you won't get the errors cause you wrote that differently

Reply:
no.. i got the error after i wrote it differently .. as an array.. 😅

Reply:
Strange!